index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="style.css" rel="stylesheet">
+        <title>Portfolio: Setup and Mobile Version Skeleton</title>
+    </head>
+    <body>
+        <header class="toolbar">
+            <span>Welcome</span>
+            <button type="button" class="btn btn-tertiary"><img src="Images/Normal-Button/Tertiary/Icons/Union.svg" alt="union icon"></button>
+        </header>
+        <section class="headline">
+            <h1>Hey there. I’m<br>Sahar<span class="h1-span">I’m a software developer</span></h1>
+            <div>I can help you build a product , feature or website Look through some 
+                of my work and experience! If you like what you see and have a project you 
+                need coded, don’t hestiate to contact me.
+            </div>
```

Comment:
- [ ] For proper semantic purposes, this text is more suited for a **`p`** tag than a generic `div` tag. Kindly wrap this text in a `p` tag, instead of `div`